mujoco_warp/_src/io.py

```diff
@@ -370,6 +397,8 @@ def put_model(mjm: mujoco.MjModel) -> types.Model:
     dof_tri_row=wp.array(dof_tri_row, dtype=int),
     dof_tri_col=wp.array(dof_tri_col, dtype=int),
     geom_type=wp.array(mjm.geom_type, dtype=int),
+    geom_sdf_plugin_type=wp.array(sdf_types, dtype=int),
```

Comment:
Do we need geom_sdf_plugin_type? I think the only information that now sdf_narrowphase_kernel needs is whether is a user supplied sdf or now.

Reply:
I guess in my mind I was thinking that each sdf would have a different function but you are right with the limitation that the user can only pass one custom sdf, we have to pass the type.